docs/rule/android_binary.soy

```diff
@@ -180,6 +180,18 @@ An <code>android_binary()</code> rule is used to generate an Android APK.
 {call android_common.deps_apk_arg /}
 
 {call buck.arg}
+  {param name : 'cpu_filters' /}
+  {param default : '[]' /}
+  {param desc}
+  The CPU architecture filter applied to the final apk. Could be a subset of <code>ARM</code>, <code>ARMV7</code>, <code>X86</code>, <code>X86_64</code>, <code>MIPS</code>.
+
+  <p>
+  <strong>Note</strong>: If you set this parameter, you must set a environment variable named <code>ANDROID_NDK</code> pointing to your local android ndk root dir, otherwise BUCK build will fail.
```

Comment:
There are actually a variety of ways to set the NDK.  However, it would be best for you to link to https://buckbuild.com/setup/install.html

Also, a nit, but you should indent inside the `<p></p>` one level.